FormalConjectures/Wikipedia/Sendov.lean

```diff
@@ -0,0 +1,71 @@
+/-
+Copyright 2025 The Formal Conjectures Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-/
+
+import FormalConjectures.Util.ProblemImports
+
+/-!
+# Sendov's conjecture
+
+*Reference:* [Wikipedia](https://en.wikipedia.org/wiki/Sendov%27s_conjecture)
+-/
+
+
+open Polynomial
+
+/-- The predicate that a polynomial satisfies the hypotheses of Sendov's conjecture.
+
+`f.IsSendov` holds if `f` has degree at least 2 and all roots of `f` lie in the unit disc
+of the complex plane. -/
+private def Polynomial.IsSendov {f : ℂ[X]} : Prop :=
```

Comment:
Is there a reason why `{f : ℂ[X]}` is implicit and not `(f : ℂ[X])`?

Reply:
None! Made the argument explicit.